lib/src/dependency.dart

```diff
@@ -43,6 +45,7 @@ class GitReference extends DependencyReference {
   final String ref;
 
   GitReference(this.url, this.ref);
+
```

Comment:
This change and the next one are just autoformatting. LMK if you want to keep the original formatting.

Reply:
That's cool. I treat darmfmt as gospel